galaxykit/collections.py

```diff
@@ -166,3 +167,17 @@ def move_collection(
             if timeout < 0:
                 raise
     return True
+
+def delete_collection(client, namespace, collection, version):
+    """
+    Delete collection version
+    """
+    
+    if version == None:
+        delete_url = f"/api/automation-hub/content/published/v3/collections/{namespace}/{collection}/"
+    else:
+        delete_url = f"content/published/v3/collections/{namespace}/{collection}/versions/{version}"
```

Comment:
is this URL correct? why doesn't it have the '/api/automation-hub/' part ?

Reply:
Hmmm this is something I should clear up in the README.

The GalaxyClient object should be initialized with the api root - this means that it automatically gets added to any url when you do a client.post or a client.delete or a client.get call.

So, changes needed for this PR are to remove `/api/automation-hub/` from all the URLs added in this PR.